lib/pbench/cli/agent/commands/stddev.py

```diff
@@ -0,0 +1,28 @@
+"""
+pbench-avg-stddev
+
+
+input: a bunch of numbers on cmd line
+output: an average (mean), sample standard deviation, sample standard deviation
+percent, and which input was closest to the average.
+
+nargs = -1, accepts an unlimited number of arguments
+"""
+
+import click
+
+import numpy as np
```

Comment:
This seems really excessive to pull in `numpy` is there something we can do that is less heavy weight?

Reply:
Done